third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -61,6 +61,10 @@ resource "google_container_node_pool" "primary_preemptible_nodes" {
     preemptible  = true
     machine_type = "n1-standard-1"
 
+    metadata {
+      disable-legacy-endpoints = "true"
```

Comment:
I assume this means the api is expecting a boolean jammed in a string? In which case :(

Reply:
This is JSON so there are no rules :)